src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest_getVisitBalances.kt

```diff
@@ -138,21 +136,10 @@ class BookingResourceIntTest_getVisitBalances : ResourceTest() {
       @Test
       fun `returns 200 if in user caseload`() {
         webTestClient.get().uri("/api/bookings/offenderNo/A1234AA/visit/balances")
-          .headers(setAuthorisation("ITAG_USER", listOf("")))
+          .headers(setAuthorisation("ITAG_USER", listOf()))
           .exchange()
           .expectStatus().isOk
       }
-
-      @Test
-      fun `invalid user access produces telemetry event`() {
```

Comment:
I didn't notice the telemetry being removed - is it never triggered now?

Reply:
no - as it was part of the test to check if we should be throwing a 403 (instead of 404)